web-filter/js/MainController.js

```diff
@@ -14,6 +14,11 @@ Ext.define('Ung.apps.webfilter.MainController', {
     getSettings: function () {
         var v = this.getView(), vm = this.getViewModel();
 
+        // set the enabled custom block page based on URL existance
+        vm.bind('{settings.customBlockPageUrl}', function (url) {
+            vm.set('settings.customBlockPageEnabled', url.length > 0);
+        });
+
```

Comment:
I don't think we should be removing the checkbox for this. I'm ok with us writing an invalid ip/hostname if the user has the checkbox disabled. We'll never use the ip value in the backend